### PR TITLE
LPAL-451 reword and rejig post environment domain message

### PR DESCRIPTION
**Pull request by williamfalconeruk**
## Purpose

Our build pipelines take some time, but the infra is ready before the end to end Cypress tests are run. 

This change gives developers the URL's earlier so they can access the ephemeral environments. The post environment domains job will run as the tests start. 

We will also adjust our repo to wait on the jobs in Cypress to complete (that is an org infra change). See: https://github.com/ministryofjustice/opg-org-infra/pull/916

Fixes LPAL-451

## Approach

- PR build now completes on the tests with the message step done at the same time
- Clarifications to post environment domains content:
    - says environment deployment rather than ready
    - Caveat added about still running tests
    - link exposed to the running workflow for convenience 

## Learning

N/A

## Checklist

* [X] I have performed a self-review of my own code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] I have added mandatory tags to terraformed resources, where possible
* [ ] If I added a package.json or composer.json, I also made sure this is included in the script in `.github/workflows/dependabot-update.yml`
* [ ] The product team have tested these changes

### CODE REVIEW

File: scripts/pipeline/slack_notifications/templates/post_environment_domains.j2 [hunk 1]
```diff
@@ -7,10 +7,24 @@
             "text":
             {
                 "type": "plain_text",
-                "text": "Online LPA Development Environment Ready",
+                "text": "Online LPA Development Environment Deployed",
                 "emoji": false
             }
-        }
+        },
+		{
```
Comment (townxelliot) [inline scripts/pipeline/slack_notifications/templates/post_environment_domains.j2:14]: Indentation is inconsistent here.

**Review by townxelliot — COMMENTED** [review]
Only a minor comment, but looks fine other than that.

**Review by townxelliot — DISMISSED** [review]

File: scripts/pipeline/slack_notifications/templates/post_environment_domains.j2 (hunk 1 above)
Comment (williamfalconeruk) [reply scripts/pipeline/slack_notifications/templates/post_environment_domains.j2:14]: i'll take a look :)

Comment (williamfalconeruk) [reply scripts/pipeline/slack_notifications/templates/post_environment_domains.j2:14]: yep  - it's placement of braces - another one of those potential things for code wars 🤣 . i'll fix and push - if you could re-approve please?

**Comment by williamfalconeruk** [comment]
Example: 
Screenshot 2021-07-15 at 14 53 19

**Review by townxelliot — APPROVED** [review]

**Pull request merged**